rsqueakvm/model/compiled_methods.py

```diff
@@ -205,8 +219,10 @@ def end_pc(self):
 
     @jit.elidable_promote()
     def fetch_bytecode(self, pc):
-        assert pc >= 0 and pc < len(self.bytes)
-        return self.bytes[pc]
+        try:
+            return self.bytes[pc]
```

Comment:
I was getting crashes with some methods, I figured if we go out of bounds, rather than crashing, we can just do a return.